supabase_py/lib/Storage/StorageBucketApi.py

```diff
@@ -0,0 +1,104 @@
+import requests
```

Comment:
please rename submodule from `Storage` to `storage`

Reply:
and rename StorageBucketApi.py to storage_bucket_api.py

Reply:
Agree, please rename all files to snake case.

Reply:
Thanks for the suggestion i did not know this.